### PR TITLE
feat(switch): upgrade to mdc-web v1

### PR DESCRIPTION
**Pull request by gugu**

### CODE REVIEW

File: packages/switch/index.tsx [hunk 1]
```diff
@@ -22,8 +22,7 @@
 
 import * as React from 'react';
 import classnames from 'classnames';
-// @ts-ignore no .d.ts file
-import {MDCSwitchFoundation} from '@material/switch/dist/mdc.switch';
+import {MDCSwitchFoundation} from '@material/switch/foundation';
```
Comment (moog16) [inline packages/switch/index.tsx]: add the MDCSwitchAdapter to the `get adapter()` method in this component please

**Comment by moog16** [comment]
@gugu please resolve conflicts :)

**Comment by gugu** [comment]
@moog16 it is fixed, but fails because of mdc-card/mdc-ripple conflict

**Comment by moog16** [comment]
@gugu these tests aren't failing on the `feat/mdcweb-typescript-update` branch, are they? It looks like you're not up to date with that branch. Can you try merging `feat/mdcweb-typescript-update` to your branch?

**Comment by moog16** [comment]
related #697

**Comment by moog16** [comment]
@gugu looks like the same issue is happening here as it is on for tab-bar. I think tests should pass after you cherry pick the commits to a new PR.

**Pull request merged**